ConservationInternational/resilienceatlas
# RA2-25: Homepages (BE part)

Adding all features required to support Homepages:
 - new Homepage model that keeps basic information about homepage
 - new HomepageJourney model that stores info about journeys shown at Homepage
 - new HomepageSection model which saves information about every individual section under journey section
 - integration with Admin which allows to create/update/delete Homepages
 - adding new API endpoint which allows to get Homepage for appropriate subdomain

## Testing instructions

I think the simplest way how to test this manually is to:
 - login to Admin and create new Homepage (add info about journeys and at least one section)
 - go to swagger doc and get homepage data via API. Check that data corresponds to data which you have used when creating new homepage

## Tracking

https://vizzuality.atlassian.net/browse/RA2-139
https://vizzuality.atlassian.net/browse/RA2-140
https://vizzuality.atlassian.net/browse/RA2-141
https://vizzuality.atlassian.net/browse/RA2-142
https://vizzuality.atlassian.net/browse/RA2-143
